libs/sdk-core/src/breez_services.rs

```diff
@@ -235,6 +235,23 @@ impl BreezServices {
     ) -> SdkResult<Payment> {
         self.start_node().await?;
         let parsed_invoice = parse_invoice(bolt11.as_str())?;
+        let invoice_amount_msat = parsed_invoice.amount_msat.unwrap_or_default();
+        let provided_amount_msat = amount_sats.unwrap_or_default() * 1000;
```

Comment:
Maybe this is a good time to change the method signature to amount_msat? I think we'd want to do that anyway?

Reply:
I agree, changing to `msat` is a step towards #417.

Reply:
I agree with both of you. I did change this method signature to msat. Not all other methods.
I think we have 2 more methods where we should change (I am not sure the onchain methods should use msat).
I suggest to do it in a separate PR.